frontend/src/state/backendApi.ts

```diff
@@ -415,6 +421,7 @@ const apiStore = {
                             this.messagesTotalConsumed = Number(res.controlMessage.value.messagesConsumed);
                             break;
                         case 'done':
+                            console.log('done phase')
```

Comment:
Can we remove the console logs? I think this would always print a log message to the console on a successful list message response. We try to keep the console log relatively lean (only weird behaviour should make it to the browser console ideally)